examples/import-export/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	c "github.com/ostafen/clover"
+)
+
+func main() {
+	db, _ := c.Open("clover-db")
```

Comment:
I think here it is useful to show how to open the database in memory, since the import/export feature is particularly relevant in that case

Reply:
I've noticed that the `c.InMemeoryMode` doesn't create a directory for the DB. What are some use-cases where this mode could be used?

Reply:
I've noticed that the `c.InMemeoryMode` doesn't create a directory for the DB. What are some use-cases where this mode could be used?

Reply:
As the name suggests, the InMemoryMode maintains documents entirely in memory. This is useful, for example, in those scenarios where you are using clover as a cache. Then you need fast retrieval performance. In those cases, you can use the Export/Import methods to store/restore the state of the cache.

Reply:
Alright, thanks for the help. I'll look into such use cases and give it a shot :)